flang/include/flang/Parser/parse-tree.h

```diff
@@ -2492,6 +2494,16 @@ struct StopStmt {
   std::tuple<Kind, std::optional<StopCode>, std::optional<ScalarLogicalExpr>> t;
 };
 
+// F2023: R1167 notify-variable -> scalar-variable
+using NotifyVariable = Scalar<Variable>;
```

Comment:
Just use `Scalar<Variable>` in the tuple below; the new type here doesn't add value.

Reply:
@klausler Thanks for the review, comment addressed.